_1327/templates/base.html

```diff
@@ -160,6 +160,15 @@
 				</ul>
 			</div>
 			<div class="login-area pull-right hidden-print">
+				{% get_current_language as LANGUAGE_CODE %}
+				{% if LANGUAGE_CODE == "de" %}
+					<span class="selected-language">DE</span> |
+					<a onclick="setLanguage('en');">EN</a>
```

Comment:
This is definitely possible but I don't understand why it would improve the code. Could you please explain it more detailed?

Reply:
This is basically about consistency. We never set the `onclick` handlers directly in HTML.
We also want to separate logic description (aka Javascript) and structure description (aka HTML).
Does that help you?

Reply:
Yes, that explains it. However, one can argue that it is better the way it is right. You can instantly see that the element is doing something when clicked. Otherwise you may miss this information if you don't accidentally look into the script section. I would rather change the rest of the code than this part.

Reply:
Fair point, but do you really want to change it **everywhere**? If so, this is definitely material for another PR and should not be included in this PR. So, you could either do it the old way now, or do it everywhere and make us merge that extra PR, before we can merge this, your call :wink:

Reply:
@janno42 what is your opinion on this topic?

Reply:
i would suggest using `onClick` here and opening an issue for changing it everywhere else.

Reply:
okay, fair enough...